queries.py

```diff
@@ -149,3 +149,38 @@ def construct_mail_query(graph_uri, email, outbox_folder_uri):
     """
     q = q.format(graph_uri, email, outbox_folder_uri)
     return q
+
+def find_kalliope_mail(bestuurseenheid_graph, max_bericht_age=7):
+    oldest = datetime.now(tz=TIMEZONE) - timedelta(days=max_bericht_age)
+    oldest = oldest.replace(microsecond=0).isoformat()
+    bestuurseenheid_graph = escape_helpers.sparql_escape_uri(bestuurseenheid_graph)
+    q = """
+        PREFIX schema: <http://schema.org/>
+        PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+        PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+        SELECT DISTINCT ?dossiernummer ?bericht ?betreft ?inhoud ?verzendDatum ?verzender ?van ?ontvanger ?ontvangDatum ?mailadres
+        WHERE {{
+            GRAPH ?g {{
+                ?conversatie a schema:Conversation;
+                    schema:identifier ?dossiernummer;
+                    schema:about ?betreft;
+                    schema:hasPart ?bericht.
+                ?bericht a schema:Message;
+                    <http://mu.semte.ch/vocabularies/core/uuid> ?uuid;
+                    schema:dateSent ?verzendDatum;
+                    schema:text ?inhoud;
+                    schema:sender ?van;
+                    schema:recipient ?naar;
+                    schema:dateReceived ?ontvangDatum;
+                    ext:heeftBehandelaar ?behandelaar.
+
+                ?behandelaar schema:email ?mailadres.
+            }}
+          ?van skos:prefLabel ?verzender.
+          ?naar skos:prefLabel ?ontvanger.
+          FILTER (?ontvangDatum > "{0}"^^xsd:dateTime)
+        }}
+        ORDER BY DESC(?verzonden)
+        LIMIT 200
```

Comment:
We miss a filter to ensure we don't process the same message multiple times

Reply:
Filter added + graph matcher that similar to regular notification query